piksi_tools/console/tracking_view.py

```diff
@@ -10,53 +10,53 @@
 # EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
 # WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
 
-from chaco.api import ArrayPlotData, Plot
+from random import randint
 import time
+
+from chaco.api import ArrayPlotData, Plot
 from chaco.tools.api import LegendTool
 from enable.api import ComponentEditor
 from pyface.api import GUI
-from sbp.tracking import SBP_MSG_TRACKING_STATE
 from traits.api import Instance, Dict, List, Int, Bool
 from traitsui.api import Item, View, VGroup, HGroup, Spring
+
 import numpy as np
 
 from piksi_tools.acq_results import SNR_THRESHOLD
 from piksi_tools.console.gui_utils import CodeFiltered
-from piksi_tools.console.utils import code_to_str, code_is_gps, code_is_glo,\
+from piksi_tools.console.utils import call_repeatedly, code_to_str,\
+                                      code_is_gps, code_is_glo,\
                                       SUPPORTED_CODES
 
+from sbp.tracking import SBP_MSG_TRACKING_STATE, SBP_MSG_TRACKING_STATE_DEP_B
+
 NUM_POINTS = 200
 TRK_RATE = 2.0
+CHANNEL_COUNT = 64
+
+color_code = lambda: int('0x%02x%02x%02x' %
+                         (randint(0,255), randint(0,255), randint(0,255)), 16)
```

Comment:
Sure it would bring some value, but at the same time a lot of people seemed to agree that in the original line graph it was very hard to distinguish between separate colors, so the value was lost there.